apps/parsley/src/types/logs.ts

```diff
@@ -22,7 +22,7 @@ interface SkippedLinesRow {
 
 interface SectionHeaderRow {
   functionID: string;
-  functionName: string;
+  functionName: string | undefined;
```

Comment:
This should technically always be defined since functions containing top level commands are not pushed to processedLogLines. The ts compiler can't infer that from [here](https://github.com/evergreen-ci/ui/pull/325/files#diff-86d2f3595501c7254caabc17d058ba3afcbf84faa7d6b9835120ef4e59fb0d36R95) and [here](https://github.com/evergreen-ci/ui/pull/325/files#diff-572ddb65042aa207cbd06f85b85d004a6a52b820af6421cd1f4c07fbed4e42f7R94).